### ISSUE TITLE
Better diff view on float display

### ISSUE THREAD

**Issue by oblitum** [wbthomason/packer.nvim#459]
### Describe the feature

Currently by using float view (`display = { open_fn = require('packer.util').float`), if I click `'d'` to see diff of some update after a `:PackerSync`, the diff will show in a preview behind the packer float.

I'm not sure what should be done to get this fixed, at first it seems to me that if the diff showed in another float bellow the packer float, something in the sense of telescope ui, that it would work better than current state.

**Comment by tsakirist** [comment]
I agree, it's really cumbersome to check for plugin updates this way.

Would be great to have another floating pop-window with just the diff.

**Comment by assistcontrol** [comment]
There is a strong trend these days toward plugins leveraging floating windows. I'm excited that packer has floating window support, but its implementation is pretty incomplete. The diff and info splits are hardcoded, and once they pull focus from the float, they don't give it back (you have to manually ^W^W to get back to the float).

Popping up another float on top of the packer window would work fine for my purposes. Could the window logic be extended to diff windows?

**Comment by krishnakumarg1984** [comment]
This should be considered a bug as it severely hampers the usability.

![image](https://user-images.githubusercontent.com/15239509/172367825-d2ed3d6f-b611-414b-a8e7-7b85adb0bd48.png)

We cannot see the diffs since they are obscured by the packer float window.
